### PR TITLE
Update Container Linux references

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
- "Container Linux" -> "CoreOS Container Linux"
- "Flatcar Linux" -> "Flatcar Container Linux"

### CODE REVIEW

File: docs/advanced/worker-pools.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Worker Pools
 
-Lokomotive on AWS, Azure, Google Cloud and Packet allows additional groups of workers to be defined and joined to a cluster. For example, add worker pools of instances with different types, disk sizes, Container Linux channels, or preemptibility modes.
+Lokomotive on AWS, Azure, Google Cloud and Packet allows additional groups of workers to be defined and joined to a cluster. For example, add worker pools of instances with different types, disk sizes, Flatcar Container Linux channels, or preemptibility modes.
```
Comment (dongsupark) [inline docs/advanced/worker-pools.md:4]: Shouldn't it be `CoreOS Container Linux`?

**Review by dongsupark — COMMENTED** [review]
Looks good in general.

Comments:

File: docs/advanced/worker-pools.md (hunk 1 above)
Comment (mauriciovasquezbernal) [reply docs/advanced/worker-pools.md:4]: I'm not sure, I think this documentation is exclusively for Flatcar Container Linux, so not reason to talk about CoreOS Container Linux in this doc.

Comment (dongsupark) [reply docs/advanced/worker-pools.md:4]: ok, that makes sense, if you say so.

**Review by dongsupark — APPROVED** [review]

**Pull request merged**